pkg/console/status/auth_status.go

```diff
@@ -0,0 +1,202 @@
+package status
+
+import (
+	"context"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	configv1 "github.com/openshift/api/config/v1"
+	configv1ac "github.com/openshift/client-go/config/applyconfigurations/config/v1"
+	configv1client "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+)
+
+type AuthenticationConditionType string
+
+const (
+	ConditionTypeDegraded    = AuthenticationConditionType("Degraded")
```

Comment:
They do indeed exist in openshift/api, but in other contexts, e.g. `OperatorStatusTypeDegraded`. I wanted to avoid confusion or possible mistake because of renaming in the future. Should I use these anyway?

Side-note, the definition of the `AuthenticationConditionType` is leftover from a previous implementation that would receive the type as an argument; it's obsolete now as it's only used internally, so if we don't use the openshift/api ones, I'd just declare some local constants.